app/views/observer/search_bar_help.html.erb

```diff
@@ -0,0 +1,10 @@
+<% @title = :search_bar_help_title .t %>
+
+<div class="max-width-text">
+  <%= :pattern_search_terms_help.tp %>
+  <%= content_tag(:p, "#{:NAMES.t} #{:SEARCHES.t}", { class: "bold" }) %>
```

Comment:
I'd consider putting the observations section first since that's the default search option and the explanation of how to do date searches is only associated with the `date` option which is only available for observation searches.  Alternatively, adding the format explanation to `created` and `modified` text would resolve that issue.

Reply:
Done.